src/lambdas/generateDownload/writeOutSecureDownloadRecord.ts

```diff
@@ -11,6 +12,7 @@ export const writeOutSecureDownloadRecord = async (parameters: {
   downloadHash: string
   zendeskId: string
 }) => {
+  logger.appendKeys({ zendeskId: parameters.zendeskId })
```

Comment:
We already have the Zendesk ID in the lambda handler, so I think you should do this there: the idea is to add data to the logger as soon as possible, so any other log lines that happen get that context.

Reply:
moved it up to the handler